common/flatpak-dir.c

```diff
@@ -7932,10 +7934,22 @@ flatpak_dir_check_parental_controls (FlatpakDir    *self,
   authorized = polkit_authorization_result_get_is_authorized (result);
 
   if (!authorized)
-    return flatpak_fail_error (error, FLATPAK_ERROR_PERMISSION_DENIED,
-                               /* Translators: The placeholder is for an app ref. */
-                               _("Installing %s is not allowed by the policy set by your administrator"),
-                               ref);
+    {
+      /* Send a metrics event so we have an idea if there are any places which
+       * don’t block app installing from the UI. If everything is implemented
+       * correctly, this event should never be submitted.
```

Comment:
Maybe add "(unless the user initiated the install from the command line)" to this comment?

Reply:
Good catch, done.